pkg/applicationserver/io/web/templates.go

```diff
@@ -273,6 +277,15 @@ func (webhookTemplate) pathToMessage(s *string) *ttnpb.ApplicationWebhookTemplat
 	}
 }
 
+func (webhookTemplate) realFormat(s string) string {
+	switch s {
+	case "pb":
+		return "grpc"
```

Comment:
Wait, what is happening here?

Reply:
Well the naming of the format in the AS frontends it's always `pb` for the binary protobuffs one. If you want it, for some reason, to be exposed in the templates as `pb`, we can rename it while loading the webhook.

Reply:
I don't understand. Where is that exactly?

I see [this](https://github.com/TheThingsNetwork/lorawan-stack/blob/master/pkg/applicationserver/io/web/format_protobuf.go#L20) and this is what I meant

Reply:
I mixed the frontend name name with the format. Fixed.